index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<section class="toolbar">
+  <nav class="toolbar logo"><img src="images/logo.png" alt=""></nav>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</section>
+<section class ="container">
+  <div class="bg-image1"></div>
+  <div class="bg-image2"></div>
+  <div class="bg-image3"></div>
+  <div class="bg-image4"></div>
+  <div class="bg-image6"></div>
+  <div class="bg-image7"></div>
+  <div class="bg-imagea"></div>
+  <div class="bg-imageb"></div>
+     <div class="container intro"><h1>I'm Victor,
+       Glad to see you!</h1></div>
+          <div class="container content">
+           <p>Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+          </div>
+     <h4 class="container social">LET'S CONNECT</h4>
+     <ul>
+       <li class="container twit"><img src="images/twitter.png" alt=""></li>
+       <li class="container linked"><img src="images/linkedin.png" alt=""></li>
+     </ul>  
+</section>
```

Comment:
- [ ] please it would be better if you use the right font, font size and same spaces exactly as in the Figma design in order to match the Design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/170547214-a333a34d-f2d4-4416-9334-87af6f63b755.png)
- Figma 
![image](https://user-images.githubusercontent.com/61790587/170547285-8efdb11a-1c27-4ca8-9502-7f8dde0e67dc.png)